### PR TITLE
Install livecd-rootfs before copying over local files

### PR DESCRIPTION
**Pull request by philroche**
This means we overwrite all of livecd-rootfs not just any non upstream directories

### CODE REVIEW

**Comment by OddBloke** [comment]
So is the issue here that, previously, livecd-rootfs is installed during the build process, overwriting what we've already put in place? Whereas now it won't be (because it's already installed)?

**Comment by rcj4747** [comment]
@OddBloke "in-target buildlivefs" will install livecd-rootfs, so yes, without this patch that is where the livecd-rootfs changes would be overwritten.

File: old-fashioned-image-build [hunk 1]
```diff
@@ -130,7 +130,14 @@ lxc exec lp-$SERIES-amd64 -- hostname standalone-lp-$SERIES-amd64
 
 # Inject the files from the current tree in the right place in the LXD
 # container.
-lxc exec lp-$SERIES-amd64 -- mkdir /usr/share/livecd-rootfs
+# First install livecd-rootfs. We will replace the bulk of the livecd-rootfs
+# code with our local copy
```
Comment (rcj4747) [inline old-fashioned-image-build]: "without this livecd-rootfs is installed by buildlivefs (below) overwriting changes"

**Comment by chrisglass** [comment]
Ah, nice! Good catch!

**Pull request merged**